extensions/legit/legit.lisp

```diff
@@ -1,7 +1,8 @@
 (defpackage :lem/legit
   (:use :cl
    :lem)
-  (:export :legit-status)
+  (:export :legit-status
+           :*prompt-for-commit-abort-p*)
```

Comment:
Is there a better way than exporting this variable and making it part of the API?
For example, how about using lem:config?
Also, I personally think that the p suffix in naming should only be used for functions or function parameters (e.g. read's recursive-p).

Reply:
I like the idea to put user-facing variables into lem:config, but this system currently has no support for documentation strings, and it has no namespace system (unlike package:\*variable\*). Is lem:config really appropriate? It stores values for stuff that we changed with a global command, such as the color theme, the font size, isearch-previous-string.